rdt-open-reader/src/main/java/io/ona/rdt/activity/RDTJsonFormActivity.java

```diff
@@ -31,7 +33,7 @@
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
-        updateLocale(this);
+        updateLocale(this, Utils.getSharedPreference(this));
```

Comment:
No need to pass `Utils.getSharedPreference(this)` here based on https://github.com/onaio/rdt-standard/pull/457/files#r478863684.